packages/augur-ui/src/modules/market-charts/components/price-history/price-history.tsx

```diff
@@ -249,7 +249,7 @@ const handleSeries = (
         id === '0' &&
         !isTrading &&
         priceTimeSeries[id][priceTimeSeries[id].length - 1].price === '0';
-      const isSelected = selectedOutcomeId && selectedOutcomeId == id;
+      const isSelected = selectedOutcomeId == id;
```

Comment:
Shouldn't `==` be `===`?

Reply:
Unless you are comparing different types, like string and integer
`"0" == 0`

Reply:
this is intentional, one is a number, the other is a string.